### PR TITLE
[zh] Fix links for contribute

### PR DESCRIPTION
**Pull request by Sea-n**
Fix link and re-sync page.

https://kubernetes.io/zh/docs/contribute/new-content/overview/ -> https://kubernetes.io/zh-cn/docs/contribute/new-content/

https://kubernetes.io/zh/docs/contribute/participating/ -> https://kubernetes.io/zh-cn/docs/contribute/participate/

### CODE REVIEW

File: content/zh-cn/docs/contribute/advanced.md [hunk 1]
```diff
@@ -21,20 +21,21 @@ to learn about more ways to contribute. You need to use the Git command line
...
 
-SIG Docs [members](/docs/contribute/participate/roles-and-responsibilities/#members) can propose improvements.
+SIG Docs [members](/docs/contribute/participate/roles-and-responsibilities/#members)
+can propose improvements.
 -->
-## 提出改进建议
+## 提出改进建议   {#propose-improvements}
 
-SIG Docs 的 [成员](/zh/docs/contribute/participate/roles-and-responsibilities/#members) 可以提出改进建议。
+SIG Docs 的 [成员](/zh-cn/docs/contribute/participate/roles-and-responsibilities/#members) 可以提出改进建议。
```
Comment (howieyuen) [inline content/zh-cn/docs/contribute/advanced.md:38]:
```suggestion
SIG Docs 的[成员](/zh-cn/docs/contribute/participate/roles-and-responsibilities/#members)可以提出改进建议。
```

File: content/zh-cn/docs/contribute/advanced.md [hunk 2]
```diff
@@ -341,7 +349,7 @@ For weekly meetings, copypaste the previous week's notes into the "Past meetings
 
 **尊重大家的时间**:
 
-- 准时开始和结束会议
+准时开始和结束会议
```
Comment (howieyuen) [inline content/zh-cn/docs/contribute/advanced.md:352]:
```suggestion
按时开始和结束会议
```

File: content/zh-cn/docs/contribute/_index.md [hunk 3]
```diff
@@ -157,9 +157,9 @@ Figure 1 outlines a roadmap for new contributors. You can follow some or all of
 图 1. 新手入门指示。
 
 图 1 概述了新贡献者的路线图。
-你可以遵从`注册`和`评审`所述的某些或全部步骤。
+你可以遵从 `注册` 和 `评审` 所述的某些或全部步骤。
```
Comment (howieyuen) [inline content/zh-cn/docs/contribute/_index.md:160]: 我觉得这里应该用双引号表示引用，同时也不需要添加额外的空格
```suggestion
你可以遵从“注册”和“评审”所述的某些或全部步骤。
```

File: content/zh-cn/docs/contribute/_index.md [hunk 4]
```diff
@@ -157,9 +157,9 @@ Figure 1 outlines a roadmap for new contributors. You can follow some or all of
...
 至此，你完成了发起 PR 的准备工作，
-可以通过`发起 PR`列出的事项实现你的贡献目标。
+可以通过 `发起 PR` 列出的事项实现你的贡献目标。
```
Comment (howieyuen) [inline content/zh-cn/docs/contribute/_index.md:162]:
```suggestion
可以通过“发起 PR” 列出的事项实现你的贡献目标。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]
Please update the links in the English original text by the way.

PS: Syntax error in two graphs mermaid version 8.13.0,  but both ok in mermaid v9.1.12

File: content/zh-cn/docs/contribute/_index.md [hunk 5]
```diff
@@ -157,9 +157,9 @@ Figure 1 outlines a roadmap for new contributors. You can follow some or all of
 图 1. 新手入门指示。
 
 图 1 概述了新贡献者的路线图。
-你可以遵从`注册`和`评审`所述的某些或全部步骤。
+你可以遵从“注册”和“评审”所述的某些或全部步骤。
 至此，你完成了发起 PR 的准备工作，
-可以通过`发起 PR`列出的事项实现你的贡献目标。
+可以通过“发起 PR” 列出的事项实现你的贡献目标。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/_index.md:162]:
```suggestion
可以通过 “发起 PR” 列出的事项实现你的贡献目标。
```

File: content/zh-cn/docs/contribute/advanced.md [hunk 6]
```diff
@@ -129,37 +131,37 @@ rotated among SIG Docs approvers.
...
+- Mentoring new contributors through their first few PRs to the docs repo.
 - Helping new contributors create the more complex PRs they need to become Kubernetes members.
--[Sponsoring contributors](/docs/contribute/advanced/#sponsor-a-new-contributor) on their path to becoming Kubernetes members.
+- [Sponsoring contributors](/docs/contribute/advanced/#sponsor-a-new-contributor) on their path to becoming Kubernetes members.
 - Hosting a monthly meeting to help and mentor new contributors.
 -->
 - 监听 [Kubernetes #sig-docs 频道](https://kubernetes.slack.com) 上新贡献者的 Issue。
-- 与 PR 管理者合作为新参与者寻找[合适的第一个 issues](https://kubernetes.dev/docs/guide/help-wanted/#good-first-issue) 。 
-- 通过前几个 PR 指导新贡献者为文档存储库作贡献。 
+- 与 PR 管理者合作为新参与者寻找[合适的第一个 issues](https://kubernetes.dev/docs/guide/help-wanted/#good-first-issue)。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/advanced.md:163]: `第一个` has a little conflict with `issues` in a viewpoint of countable number.
```suggestion
- 与 PR 管理者合作为新参与者寻找[合适的第一个 issue](https://kubernetes.dev/docs/guide/help-wanted/#good-first-issue)。
```

**Review by windsonsea — COMMENTED** [review]
two `good first` concerns :)

File: content/zh-cn/docs/contribute/_index.md (hunk 5 above)
Comment (Sea-n) [reply content/zh-cn/docs/contribute/_index.md:162]: I am prefer to add a space before `“”` too. But in this project, it seems not encouraged to add that space.

CC @tengqm could you explain more on this?

**Pull request merged**